service/src/main/java/uk/gov/mca/beacons/api/export/xlsx/backup/BackupXlsxExporter.java

```diff
@@ -59,11 +57,15 @@ public void backup() throws IOException {
       return;
     }
 
-    backupToXlsxJobManager.backup(
-      fileSystemRepository.getNextExportDestination(
-        ExportFileNamer.FileType.EXCEL_SPREADSHEET,
-        BeaconsDataWorkbookRepository.OperationType.BACKUP
-      )
-    );
+    try {
+      backupToXlsxJobManager.backup(
+        fileSystemRepository.getNextExportDestination(
+          ExportFileNamer.FileType.EXCEL_SPREADSHEET,
+          BeaconsDataWorkbookRepository.OperationType.BACKUP
+        )
+      );
+    } catch (Exception e) {
+      Exception ex = e;
```

Comment:
Not doing anything with e here, maybe add a log.

Also creating a copy of what you already have which doesn't really make sense.